src/index.js

```diff
@@ -586,6 +586,21 @@ export async function martechLazy() {
 
   if (!config.personalization) {
     await loadAndConfigureAlloy(config.alloyInstanceName, alloyConfig);
+  } else if (!config.performanceOptimized) {
+    const renderDecisionResponse = await sendEvent({ renderDecisions: true, decisionScopes: ['__view__'] });
```

Comment:
Atomic render decisions call for the top view, followed by prehiding removal and reporting of the page view event

Reply:
In `await sendEvent({ renderDecisions: true, decisionScopes: ['__view__'] });`  having `renderDecisions:true` will turn send the notification automatically after the propositions were executed. Thus no need to trigger manually another display notification.